### PR TITLE
Feature/116745 tsm tooltips

### PR DESCRIPTION
**Pull request by jesseeproductions**
Start with this PR:
https://github.com/moderntribe/tribe-common/pull/816

Second PR:
https://github.com/moderntribe/event-tickets/pull/906

Last PR
https://github.com/moderntribe/event-tickets-plus/pull/653

### CODE REVIEW

File: src/resources/js/tribe-common.js [hunk 1]
```diff
@@ -135,6 +136,27 @@ tribe_auto_sysinfo.ajax = {
 
 	};
 
+	my.init_tooltip = function () {
+
+		$( document ).on( 'click', '.tribe-tooltip', function () {
+
+			var $tooltip = $( this ).closest( '.tribe-tooltip' );
+			var add = true;
+			if ( $( $tooltip ).hasClass( 'active' ) ) {
```
Comment (Camwyn) [inline src/resources/js/tribe-common.js]: `$tooltip` is already a jQuery var - no need to wrap it again?

You could also shorten this to:
`var add = $tooltip.hasClass( 'active' );`
But this is fine too 😄

**Comment by jesseeproductions** [comment]
Updated coding and minified.
